Serialization/Serialization/Helper.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Xml.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Runtime.Serialization;
+
+namespace Serialization
+{
+    public static class Helper
+    {
+        private static XmlSerializer xmlFormatter;
+        private static BinaryFormatter binaryFormatter;
+
+        static Helper()
+        {
+            xmlFormatter = new XmlSerializer(typeof(Student));
+            binaryFormatter = new BinaryFormatter();
+            
+        }
+
+        public static void SerializeToXml(Student student, Stream stream)
+        {
+            using (stream)
```

Comment:
you creating stream somewhere not in this class but dispose you executing here by using, why?

Reply:
you have to use using only one time

Reply:
My fault, fixed in a2b8f77f25c975083ebee2c718ad445e5576ff02